providers/ms365/resources/users.go

```diff
@@ -826,7 +826,7 @@ func (a *mqlMicrosoftUser) authenticationRequirements() (*mqlMicrosoftUserAuthen
 		return nil, transformError(err)
 	}
 
-	mqlAuthRequirements, err := CreateResource(a.MqlRuntime, "microsoft.userAuthenticationRequirements",
+	mqlAuthRequirements, err := CreateResource(a.MqlRuntime, "microsoft.user.AuthenticationRequirements",
```

Comment:
can we use the lr-generated string const instead? that would prevent this from happening again